src/spec/expectations.cr

```diff
@@ -385,6 +385,11 @@ module Spec
         raise ex
       end
 
+      # `NestingSpecError` is also treated.
+      if ex.is_a?(Spec::NestingSpecError) && klass != Spec::NestingSpecError
```

Comment:
What is this for?

Reply:
See full code:

```crystal
    def expect_raises(klass : T.class, message = nil, file = __FILE__, line = __LINE__) forall T
      yield
    rescue ex : T
      # We usually bubble Spec::AssertaionFailed, unless this is the expected exception
      if ex.is_a?(Spec::AssertionFailed) && klass != Spec::AssertionFailed
        raise ex
      end

      # `NestingSpecError` is treated as the same above.
      if ex.is_a?(Spec::NestingSpecError) && klass != Spec::NestingSpecError
        raise ex
      end

      # ...
    end
```

Please tell me more correct sentence which means this if you know.